### PR TITLE
linux-hikey_git: switch hikey to 4.11-rc5 kernel

### PR DESCRIPTION
**Pull request by mike-scott**
This is the first device to begin using a unified kernel base.

As a part of that we need a distro.config to be shared across devices
and remove the existing pre-set defconfig (use the default arm64
defconfig).

Signed-off-by: Michael Scott <michael.scott@linaro.org>

### CODE REVIEW

File: recipes-kernel/linux/linux-hikey_git.bb [hunk 1]
```diff
@@ -2,18 +2,19 @@ require linux.inc
 
 DESCRIPTION = "96boards-hikey kernel"
 
-PV = "4.9+git${SRCPV}"
-SRCREV_kernel = "fe42635ce7dbb6250a3960eebd8880a998c1f8c6"
+PV = "4.11-rc5+git${SRCPV}"
+SRCREV_kernel = "e83259bbe35822d20932139b47708f9edc703d7d"
```
Comment (mike-scott) [inline recipes-kernel/linux/linux-hikey_git.bb]: Not sure if we can disable this SRCREV check.  It's going to be a pain to keep updating.

File: recipes-kernel/linux/files/distro.config [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+CONFIG_CFG80211_WEXT=y
+CONFIG_MAC80211=y
+CONFIG_MAC80211_DEBUGFS=y
+CONFIG_BT_HCIUART_QCA=y
+CONFIG_BT_LE=y
+CONFIG_6LOWPAN=m
+CONFIG_6LOWPAN_DEBUGFS=y
+CONFIG_BT_6LOWPAN=m
+
+CONFIG_LOCALVERSION_AUTO=n
```
Comment (mike-scott) [inline recipes-kernel/linux/files/distro.config]: Hrm.  I left this in to make dev easier.  But maybe we should drop CONFIG_LOCALVERSION_AUTO=n
Debate among yourselves.

Comment (rsalveti) [inline recipes-kernel/linux/files/distro.config]: Yeah, probably better to remove.

File: recipes-kernel/linux/linux-hikey_git.bb (hunk 1 above)
Comment (rsalveti) [inline recipes-kernel/linux/linux-hikey_git.bb]: While it's a pain during dev, it's the only way to have reproducible builds.

**Pull request closed without merging**